modules/ROOT/pages/troubleshoot-service-binding-issues.adoc

```diff
@@ -0,0 +1,115 @@
+= Troubleshooting Service Binding Issues
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, service binding issues, troubleshooting
+
+After you create the API, you then bind your service with the API. This section provides possible causes for issues that might occue when you bind your service, and describes how to troubleshoot these errors:
+
+* <<dns-name-error,DNS Name Error>>
+* <<references non-existent-instance-error,References non-existent instance error>>
+* <<missing-entitlements-in-license,Missing entitlements in license>>
+
+[[dns-name-error]]
+== DNS Name Error
+
+If you provide a DNS name that does not comply with the DNS subdomain standard format, an error occurs and the binding fails.
+
+=== Cause
+
+This issue occurs if you provided a DNS name that does not follow the DNS subdomain standards.
+
+=== Diagnose
+
+To diagnose this issue, run one of the following commands:
++
+`$ ./service-mesh api binding list` Or `$ ./service-mesh adapter binding list`.
++
+The output of this command displays a Failed status of the binding:
++
+`The ServiceBinding "customer-API" is invalid: metadata.name: Invalid value: "customer-API": a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')`.
+
+=== Solution
+
+Ensure that you provide DNS name values that follow the DNS subdomain standards.
+
+[[references non-existent-instance-error]]
+== References Non-existent Instance Error
+
+When creating a binding, the binding command requires you to provide a name for the service and the name of the adapter that you created previously. Errors can occur if you did not specify the correct parameters for the binding command.
```

Comment:
```suggestion
When you create a binding, you must include in the binding command a name for the service and the name of the adapter that you created previously. Otherwise, you receive this error message.
```

Reply:
Done! Thanks!